jlm/rvsdg/region.cpp

```diff
@@ -66,6 +66,12 @@ argument::argument(
   }
 }
 
+argument &
+argument::CopyTo(rvsdg::region & region, jlm::rvsdg::structural_input * input)
+{
+  return *argument::create(&region, input, Type());
```

Comment:
Replacing `Type()` with `port()` here should fix the HLS test failure.

Reply:
Thanks for looking into this. This saved me a lot of time. Appreciated!